crates/ui/src/components/label/label_like.rs

```diff
@@ -97,6 +98,7 @@ impl RenderOnce for LabelLike {
                 )
             })
             .map(|this| match self.size {
+                LabelSize::Large => this.text_ui_lg(),
```

Comment:
I added this to make the title larger, but then decided not to use it. Regardless, it seemed useful and I already fixed the documentation so I thought I'd leave it :)